src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
```

Comment:
We are using this style guide over all of our projects: https://kowainik.github.io/posts/2019-02-06-style-guide
So by its rules, here we would have 7 spaces before the open bracket.